src/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestsManifestNegative.cs

```diff
@@ -42,6 +42,7 @@ private static string GetResourceStringFromReflection(string key)
         /// For NuGet EventSources we validate both "runtime" and "validation" behavior
         /// </summary>
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Full Framework doesn't have System.SR type which is used in this test.")]
```

Comment:
I've updated the PR to get the string from Environment.GetResourceString via reflection on desktop, verified locally and the test passes.